lib/plugins/config/style.css

```diff
@@ -1,159 +1,162 @@
-#config__manager div.success,
-#config__manager div.error,
-#config__manager div.info {
+#config__manager p.highlight {
+  background-color: #ccf;
+  background-image: url(../images/info.png);
+  border-color: #ccf;
   background-position: 0.5em;
-  padding: 0.5em;
-  text-align: center;
+  padding: 0.5em .5em .5em 32px;
+  /*text-align: center;*/
 }
 
-#config__manager fieldset {
-  margin: 1em;
-  width: auto;
-  margin-bottom: 2em;
-  background-color: __background_alt__;
+#config__manager {
+  clear: both;
   color: __text__;
-  padding: 0 1em;
-}
-[dir=rtl] #config__manager fieldset {
-    clear: both;
-}
-#config__manager legend {
-    font-size: 1.25em;
 }
 
-#config__manager form { }
-#config__manager table {
-    margin: 1em 0;
-    width: 100%;
+/* headlines */
+#config__manager h3 {
+  font-size: 1.75em;
+}
+#config__manager h3 {
+  font-size: 1.5em;
+  margin: 1em 0;
 }
 
-#config__manager fieldset td {
-    text-align: left;
+/* list */
+#config__manager dl {
+  background-color: __background__;
+  border: __border__ solid 1px;;
+  margin: .25rem 0;
+  display: grid;
+  grid-template-columns: 1fr 2fr;
+  line-height: 1.5em;
 }
-[dir=rtl] #config__manager fieldset td {
-    text-align: right;
+#config__manager dl dt {
+  font-size: smaller;
+  padding: .25em;
 }
-#config__manager fieldset td.value {
-    /* fixed data column width */
-    width: 31em;
+#config__manager dl dd {
+  margin: 0;
+  padding: .25em;
 }
 
-[dir=rtl] #config__manager label {
-    text-align: right;
+/* status field */
+#config__manager dl dd.status {
+  text-align: right;
 }
-[dir=rtl] #config__manager td.value input.checkbox {
-    float: right;
-    padding-left: 0;
-    padding-right: 0.7em;
+#config__manager dl dd.status ul {
+  list-style: none inside;
+  display: inline-block;
 }
-[dir=rtl] #config__manager td.value label {
-    float: left;
+#config__manager dl dd.status ul li {
+  display: inline;
+  margin: 0; padding: 0;
 }
-
-#config__manager td.label {
-    padding: 0.8em 0 0.6em 1em;
-    vertical-align: top;
+#config__manager dl dd.status ul li::after {
+  content: ',';
 }
-[dir=rtl] #config__manager td.label {
-    padding: 0.8em 1em 0.6em 0;
+#config__manager dl dd.status ul li:last-child::after {
+  content: '';
 }
-
-#config__manager td.label label {
-    clear: left;
-    display: block;
+#config__manager dl dd.status ul li::before {
+  content: '';
+  display: inline-block;
+  width: 1em; height: 1em;
+  background: transparent url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'%3E%3Ctitle%3Ehelp-circle-outline%3C/title%3E%3Cpath d='M11,18H13V16H11V18M12,2A10,10 0 0,0 2,12A10,10 0 0,0 12,22A10,10 0 0,0 22,12A10,10 0 0,0 12,2M12,20C7.59,20 4,16.41 4,12C4,7.59 7.59,4 12,4C16.41,4 20,7.59 20,12C20,16.41 16.41,20 12,20M12,6A4,4 0 0,0 8,10H10A2,2 0 0,1 12,8A2,2 0 0,1 14,10C14,12 11,11.75 11,15H13C13,12.75 16,12.5 16,10A4,4 0 0,0 12,6Z' style='fill:%23CCC'/%3E%3C/svg%3E") center no-repeat;
```

Comment:
What are all those inline SVGs? Do we really need to have them here? why not inline them as proper elements in the HTML?

Reply:
The inline SVGs replace separate files, which would otherwise need to be loaded each separately by the browser. Inlining them here reduces the total load time.